distributed/shuffle/_shuffle.py

```diff
@@ -207,19 +207,19 @@ def _cull(self, parts_out: Iterable[int]) -> P2PShuffleLayer:
             parts_out=parts_out,
         )
 
-    def _keys_to_parts(self, keys: Iterable[_T_Key]) -> set[int]:
+    def _keys_to_parts(self, keys: Iterable[Key]) -> set[int]:
         """Simple utility to convert keys to partition indices."""
         parts = set()
         for key in keys:
             if isinstance(key, tuple) and len(key) == 2:
-                _name, _part = key
-                if _name != self.name:
-                    continue
-                parts.add(_part)
+                name, part = key
+                if part == self.name:
```

Comment:
Is this change intentional?

Reply:
yes (mostly cosmetic). The necessary bit is the assertion

Reply:
Thx!

Reply:
whoops. Saw it now. Thanks for spotting!